plugins/src/core/push.ts

```diff
@@ -0,0 +1,7 @@
+export interface PushNotification {
+  id: number;
+  title?: string | null;
+  message?: string | null;
+  data?: { [key: string]: any } | null;
```

Comment:
same as above. consistency with cordova ios side/Flutter, verify it's optional VS null